src/Uno.UI.Runtime.Skia.X11/X11WindowWrapper.cs

```diff
@@ -42,7 +43,7 @@ public override string Title
 		}
 	}
 
-	public override object NativeWindow => _host.RootX11Window;
+	public override object NativeWindow => new X11NativeWindow(_host.RootX11Window.Window);
```

Comment:
Would this be a breaking change from 5.2?

Reply:
Hmm, actually, yes. I don't think anyone dependend on NativeWindow having a certain layout (especially since there is no documentation that mentions how to deal with NativeWindow on X11 and that NativeWindow is only supported on "some platforms", so the reasonable implementation would be that it's not supported), but theoretically, yes. It's a breaking change.